core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -333,8 +346,9 @@
       "Maximum time manager will wait for tserver available threshold "
           + "to be reached before continuing. When set to 0 or less, will block "
           + "indefinitely. Default is 0 to block indefinitely. Only valid when tserver available "
-          + "threshold is set greater than 0. Added with version 1.10"),
-  // properties that are specific to tablet server behavior
+          + "threshold is set greater than 0. Added with version 1.10"), // properties that are
+                                                                         // specific to tablet
+                                                                         // server behavior
```

Comment:
It seems this comment may have been meant as a separator between MANAGER and the TSERV properties below. It seems that just moving it down one line would be appropriate